src/Schema/Domain/Content/NameHelper.php

```diff
@@ -90,7 +96,14 @@ public function domainGroupField(ContentTypeGroup $contentTypeGroup)
 
     public function fieldDefinitionField(FieldDefinition $fieldDefinition)
     {
-        return lcfirst($this->toCamelCase($fieldDefinition->identifier));
+        $fieldName = lcfirst($this->toCamelCase($fieldDefinition->identifier));
+
+        // Workaround for https://issues.ibexa.co/browse/EZP-32261
+        if (\array_key_exists($fieldName, $this->fieldNameOverrides)) {
+            return $this->fieldNameOverrides[$fieldName];
```

Comment:
IMHO is worth to warn user at this point that name collision happened.

Reply:
Added a warning level log entry so it will be shown in the output of
```
php bin/console ezplatform:graphql:generate-schema
```